src/types/api.ts

```diff
@@ -84,3 +85,52 @@ export interface AddOrUpdateUserRequest {
 }
 
 export type AddOrUpdateUserResponse = ApiResponse<{ address: string } | null>;
+
+/**
+ * Response for a created hyperboard
+ */
+export type HyperboardCreateResponse = ApiResponse<{
+  id: string;
+} | null>;
+
+/**
+ * Interface for creating a hyperboard
+ */
+export interface HyperboardCreateRequest {
+  chainIds: number[];
+  title: string;
+  collections: {
+    title: string;
+    description: string;
+    hypercerts: {
+      hypercertId: string;
+      factor: number;
+    }[];
+  }[];
+  backgroundImg?: string;
+  borderColor: string;
+  adminAddress: string;
+  signature: string;
+}
+
+/**
+ * Interface for updating a hyperboard
+ */
+export interface HyperboardUpdateRequest {
```

Comment:
implements `HyperboardCreateRequest` or something similar as the types fully overlap except for `id`

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/commit/9436dd6af93162b30d97951db3cc68888e364666